hackathon_site/hardware/api_filters.py

```diff
@@ -0,0 +1,24 @@
+from django_filters import rest_framework as filters
+from .models import Hardware
+from .serializers import HardwareSerializer
+from django_filters.fields import Lookup
+
+
+class HardwareFilter(filters.FilterSet):
+    queryset = Hardware
+    serializer_class = HardwareSerializer
+
+    class Meta:
+        model = Hardware
+        fields = ["name", "quantity_available"]
+
+    def comma_separate_filter(self, queryset, value):
+        value_list = value.split(",")
+        queryset = super().filter(queryset, Lookup(value_list, "in"))
+
+        return queryset
+
+    in_stock = filters.BooleanFilter(
```

Comment:
For both the filters (`in_stock` and `ids`, can you add a `label` parameter to the filter, which will give it a name in the DRF browser? `"In Stock" and "IDs" will work. You can also add a `help_text` if you want, explaining how to use the field.

Reply:
Thank you so much Graham for your detailed explanation!!! I have learnt a lot from them! @grahamhoyes